src/server/protocol.d.ts

```diff
@@ -194,6 +194,14 @@ declare namespace ts.server.protocol {
     }
 
     /**
+      * Go to implementation request; value of command field is
+      * "implementation". Return response giving the file locations that
+      * implement the symbol found in file at location line, col.
+      */
+    export interface ImplementationRequest extends FileLocationRequest {

```

Comment:
I was following what `TypeDefinitionRequest` and `DefinitionRequest` did (see above).